masonry/src/passes/accessibility.rs

```diff
@@ -91,7 +91,7 @@ fn build_accessibility_tree(
 // --- MARK: BUILD NODE ---
 fn build_access_node(widget: &mut dyn Widget, ctx: &mut AccessCtx) -> Node {
     let mut node = Node::new(widget.accessibility_role());
-    node.set_bounds(to_accesskit_rect(ctx.widget_state.window_layout_rect()));
```

Comment:
I don't think this is correct. This should be the "layout" rect, not the bounding rect.

Reply:
Hmm, reading the docs of accesskit, I think there's more to it, as the node can be transformed. I've pushed (untested) code how I think it needs to be handled according to the docs, I currently don't have a setup to quickly evaluate this. Maybe you can check again? I'm not sure though whether the scroll translation should be applied though...